EnableJupyterDisplay.swift

```diff
@@ -0,0 +1,117 @@
+import Cryptor
+import Foundation
+
+struct Header: Codable {
+    let msg_id: String
```

Comment:
you could use https://developer.apple.com/documentation/foundation/jsonencoder/keyencodingstrategy/converttosnakecase so that you can name these fields with the proper swift camelCase convention